profiling_router.go

```diff
@@ -0,0 +1,146 @@
+package handlers
+
+import (
+	"context"
+	"net/http"
+	"net/http/pprof"
+	"os"
+	"strconv"
+
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/go-utils/logger"
+)
+
+const PprofRoutePrefix = "/debug/pprof"
+
+type profiling struct {
+	Enable bool
+	Auth   authentication
+}
+
+type authentication struct {
```

Comment:
`authentication` seems too generic

```suggestion
type pprofAuthentication struct {
```